HerPublicWebsite.BusinessLogic/Services/RegularJobs/DateHelper.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace HerPublicWebsite.BusinessLogic.Services.RegularJobs;
+
+public class DateHelper
+{
+    private readonly Func<DateTime> getToday;
+    
+    public DateHelper(): this(() => DateTime.Today)
+    {
+    }
+
+    public DateHelper(Func<DateTime> getToday)
+    {
+        this.getToday = getToday;
+    }
+
+    public DateTime GetStartOfPreviousMonth()
+    {
+        var today = getToday();
+        var oneMonthAgo = today.AddMonths(-1);
+        return oneMonthAgo.AddDays(1 - oneMonthAgo.Day); // Remove a month, and set day to 1st
```

Comment:
This comment is out of date. The month removal is on the previous line now.

You do not need a comment for the month line, since that is clear from the method call and variable name, but it might be worth leaving the `// Set day to 1st` part.